### PR TITLE
attempt to teach travis to unittest python

### PR DESCRIPTION
**Pull request by DanielCollins**

### CODE REVIEW

**Comment by DanielCollins** [comment]
hey neat, it works.

File: python/tests/test_everything.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def test_test():
+    pass
```
Comment (DanielCollins) [inline python/tests/test_everything.py]: todo: real unit tests go here

Comment (Amtron) [reply python/tests/test_everything.py]: @DanielCollins Did you want to add these tests in another PR?

Comment (DanielCollins) [reply python/tests/test_everything.py]: yes

Comment (DanielCollins) [reply python/tests/test_everything.py]: here i just demonstrate that testing works

**Pull request merged**